### PR TITLE
Fix:  order api query filter fixed #818

### LINKED ISSUE #818: Order fetching issues with REST API
**These doesn't work as expected:**

```
wp-json/dokan/v1/orders/?status=processing
wp-json/dokan/v1/orders/?order=asc&orderby=id
wp-json/dokan/v1/orders/?exclude=3589
```

### PR DESCRIPTION
**Pull request by zamandevs**
fixed #818

### CODE REVIEW

File: includes/Order/functions.php [hunk 1]
```diff
@@ -49,8 +49,16 @@ function dokan_get_seller_amount_from_order( $order_id, $get_array = false ) {
  *
  * @return array
  */
-function dokan_get_seller_orders( $seller_id, $status = 'all', $order_date = NULL, $limit = 10, $offset = 0, $customer_id = null ) {
+function dokan_get_seller_orders( $request ) {
```
Comment (kapilpaul) [inline includes/Order/functions.php:52]: Please update the doc block.

File: includes/Order/functions.php [hunk 2]
```diff
@@ -74,8 +86,9 @@ function dokan_get_seller_orders( $seller_id, $status = 'all', $order_date = NUL
                 p.post_status != 'trash'
                 {$date_query}
                 {$status_where}
+                {$exclude}
             GROUP BY do.order_id
-            ORDER BY p.post_date DESC
+            ORDER BY {$order_by} $order
```
Comment (kapilpaul) [inline includes/Order/functions.php:91]: wrap $order in a curly brace eg. {$order}

File: includes/Order/functions.php (hunk 1 above)
Comment (kapilpaul) [inline includes/Order/functions.php:52]: This param changes will give an error in https://github.com/weDevsOfficial/dokan/blob/d8cf54a6fef803a5ff97fd0b010ff00cb4c7a15f/includes/Dashboard/Templates/Orders.php#L112

&

https://github.com/weDevsOfficial/dokan/blob/d8cf54a6fef803a5ff97fd0b010ff00cb4c7a15f/includes/Dashboard/Templates/Orders.php#L126

**Comment by sabbir1991** [comment]
Do not change the function params.

**Pull request closed without merging**